### PR TITLE
Dan's fork of the Danebook pages

### PR DESCRIPTION
**Pull request by dsberger**

### CODE REVIEW

File: WHY.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.navbar {
```
Comment (daau) [inline WHY.txt:1]: Not sure what this file is for..?

File: about.html [hunk 2]
```diff
@@ -1 +1,187 @@
-<!-- Your markup here! -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (daau) [inline about.html:5]: Good use of meta tags

File: about.html [hunk 3]
```diff
@@ -1 +1,187 @@
...
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+
+    <nav class="navbar navbar-default navbar-fixed-top">
+      <div class="container">
```
Comment (daau) [inline about.html:21]: Also a possibility: container-fluid

File: about.html [hunk 4]
```diff
@@ -1 +1,187 @@
...
+          </form>
+          <ul class="nav navbar-nav navbar-right">
+            <li><a href="#">Harry</a></li>
+          </ul>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
+    </nav>
+
+    <header>
+      <div class="jumbotron">
```
Comment (daau) [inline about.html:51]: Might not be the most appropriate class, as it leaves a grey background that's not included in the spec. Looks cool, though!

File: about.html [hunk 5]
```diff
@@ -1 +1,187 @@
...
+      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+
+    <nav class="navbar navbar-default navbar-fixed-top">
+      <div class="container">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
```
Comment (daau) [inline about.html:23]: Nice!

File: about.html [hunk 6]
```diff
@@ -1 +1,187 @@
...
+        </div>
+      </div>
+    </header>
+
+    <main id="about" class="container">
+      <div class="row section-banner">
+        <div class="col-md-2 col-md-offset-5 section-header">
+          <h1>About</h1>
+        </div>
+        <div class="col-md-2 col-md-offset-3 edit-button">
```
Comment (daau) [inline about.html:83]: Noticed that button effect. Nice touch!

File: about.html [hunk 7]
```diff
@@ -1 +1,187 @@
...
+              consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+              cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+              proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
```
Comment (daau) [inline about.html:185]: This was a well organized HTML file, but I'd add more comments in the future to help discern components.

It's pretty clear you guys spent a lot of time on this. I'd be proud of this code.

File: about_edit.html [hunk 8]
```diff
@@ -1 +1,179 @@
...
+  </head>
+  <body>
+
+    <nav class="navbar navbar-default navbar-fixed-top">
+      <div class="container">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
```
Comment (daau) [inline about_edit.html:26]: Some redundant code here

File: about_edit.html [hunk 9]
```diff
@@ -1 +1,179 @@
...
+          </form>
+          <ul class="nav navbar-nav navbar-right">
+            <li><a href="#">Harry</a></li>
+          </ul>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
+    </nav>
+
+    <header>
+      <div class="jumbotron">
```
Comment (daau) [inline about_edit.html:51]: Again, use of jumbotron might not be appropriate here

File: about_edit.html [hunk 10]
```diff
@@ -1 +1,179 @@
...
+        <div id="photos-panel" class="container">
+          <img src="img/hogwarts_small.jpg" alt="User Cover Photo" id="cover-photo">
+          <img id="profile-photo" src="img/harry_potter_small.jpg" alt="User Profile Photo" height="200px">
+          <a id="user-name">Harry Potter</a>
```
Comment (daau) [inline about_edit.html:55]: Nice use of CSS here.

File: about_edit.html [hunk 11]
```diff
@@ -1 +1,179 @@
...
+              <div class="col-xs-8">
+                <input type="text" value="Hogwart's College">
+              </div>
+            </div>
+            <div class="row extra-line-height">
+              <div class="col-xs-4">
+                <strong>Hometown:</strong>
+              </div>
+              <div class="col-xs-8">
+                <input type="text" value="Godrick's Hollow">
```
Comment (daau) [inline about_edit.html:111]: There should be margin-top and margin-bottom for the input text fields

File: about_edit.html [hunk 12]
```diff
@@ -1 +1,179 @@
...
+            <div class="row extra-line-height">
+              <div class="col-xs-4">
+                <strong>Telephone:</strong>
+              </div>
+              <div class="col-xs-8">
+                <input type="tel" value="555-123-4567">
+              </div>
+            </div>
+          </div>
+          <div class="col-md-5 col-md-offset-1">
```
Comment (daau) [inline about_edit.html:144]: Columns might not add up to 12 (col-md-4 col-md-offset-1 and col-md-5 col-md-offset-1). I'm not sure how much of an issue that is, but it could affectm mobile layouts.

File: friends.html [hunk 13]
```diff
@@ -1 +1,177 @@
...
+            </li>
+            <li role="presentation" class="col-md-offset-3">
+              <a href="about_edit.html">Edit Profile</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+
+
```
Comment (daau) [inline friends.html:78]: Nicely organized components here (compared to previous two files)

File: friends.html [hunk 14]
```diff
@@ -1 +1,177 @@
...
+              <p class="friend-count">432 friends</p>
+            </div>
+            <div class="col-md-3 unfriend">
+              <button>Unfriend me</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```
Comment (daau) [inline friends.html:174]: I really appreciate the CSS classes used here. They described the content perfectly, and were used sparingly.

File: homepage.html [hunk 15]
```diff
@@ -0,0 +1,186 @@
...
+            <div class="form-group">
+              <input type="email" class="form-control" placeholder="Email">
+              <input type="password" class="form-control" placeholder="Password">
+            </div>
+            <button type="submit" class="btn btn-default">LOG IN</button>
+          </form>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
+    </nav>
+
```
Comment (daau) [inline homepage.html:55]: Looks like there's a bug here, in Chrome. Some of the main page content is hidden under the navbar.

File: homepage.html [hunk 16]
```diff
@@ -0,0 +1,186 @@
...
+        <div class="sign-up-form col-sm-5">
+          <div class="row">
+            <div class="col-md-12">
+              <h1>Sign up</h1>
+            </div>
+          </div>
+          <form>
+            <div class="row">
+              <div class="col-md-6" id="first-name">
+                <input class="textfield" type="text" placeholder="First name">
```
Comment (daau) [inline homepage.html:77]: Some of the textfields need some margin on the top and bottom

File: homepage.html [hunk 17]
```diff
@@ -0,0 +1,186 @@
...
+              </div>
+              <div class="col-md-6">
+                <input class="textfield" type="text" placeholder="Last name">
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-md-12">
```
Comment (daau) [inline homepage.html:84]: 12 width column might not be appropriate for a text field that doesnt take up the entire width of the row

File: homepage.html [hunk 18]
```diff
@@ -0,0 +1,186 @@
...
+                  <option value="1994">1994</option>
+                  <option value="1993">1993</option>
+                  <option value="1992">1992</option>
+                  <option value="1991">1991</option>
+                </select>
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-md-12">
+                <input type="radio" name="sex" value="Female"> Female
```
Comment (daau) [inline homepage.html:171]: Some padding is needed for the radio buttons

File: photos.html [hunk 19]
```diff
@@ -1 +1,229 @@
...
+        <div class="col-md-2 col-md-offset-3 edit-button">
+          <button>Add photo!</button>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
```
Comment (daau) [inline photos.html:93]: I noticed some issues with the captions not matching the width of the photos exactly on various window sizes. This can be fixed with a simple change to the width in the CSS, to 100% of the parent class (or adjusting the px values).

File: photos.html [hunk 20]
```diff
@@ -1 +1,229 @@
...
+        </div>
+        <div class="col-md-3 captioned-photo">
+          <a href="#">
+            <img src="img/icon_photo_small.png" class="img-responsive">
+          </a>
+          <div class="caption">
+            Uploaded Jan. 1, 2015
+          </div>
+        </div>
+        <div class="col-md-3 captioned-photo">
```
Comment (daau) [inline photos.html:215]: The margin on the right is larger than the margin on the left. This is probably because the images don't fit the container 100%, and are aligned to the top-left corner of its container, resulting in a larger right margin.

Suggested fix: min-width and min-height to 100%

File: styles.scss [hunk 21]
```diff
@@ -0,0 +1,204 @@
+$background-blue: #2B78E4;
```
Comment (daau) [inline styles.scss:1]: Good use of variables here, though I'd give it a better variable name (maybe "danebook-blue", to denote the official colour)

File: timeline.html [hunk 22]
```diff
@@ -1 +1,395 @@
...
+                        <button>Comment</button>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </article>
+        </div>
+      </div>
```
Comment (daau) [inline timeline.html:389]: I can tell you guys spent a lot of time on this file. The HTML on this is very good.
